tools/wpt/virtualenv.py

```diff
@@ -92,8 +92,12 @@ def working_set(self):
 
     def activate(self):
         path = os.path.join(self.bin_path, "activate_this.py")
-        with open(path) as f:
```

Comment:
Could we just bring in `open` from `io` and use that? E.g. see https://stackoverflow.com/a/10975371/12838787

Reply:
Do you mean to simply code to "open(path, _encoding='utf-8')"_ and get rid of the condition _"sys.version_info.major >= 3"_?

Reply:
Correct; I assume we want to open in utf-8 in both versions?

Reply:
>  If I understand correctly, default encoding in python 2 is ASCII.
Not sure we want open in utf-8 in python 2. Do please correct me if I have missed anything here.

Reply:
Sorry, I overlooked that sentence completely 🤦 

So, why do we want to open in utf-8 in Python 3, if we're opening in ASCII in Python 2? Shouldn't we be consistent in what data we're going to accept here?